tests/west/project/test_project.py

```diff
@@ -39,7 +40,10 @@ def cmd(cmd):
 
     # cmd() takes the command as a string, which is less clunky to work with.
     # Split it according to shell rules.
-    split_cmd = shlex.split(cmd)
+    if sys.platform == 'win32':
+        split_cmd = shlex.split(cmd, posix=False)
```

Comment:
A comment explaining why this is here would be nice:

```python
# posix=False prevents backslashes from getting stripped from paths on Windows
```

Wonder if we could just pass `posix=False` for both. `shlex.split()` it works the same regardless of platform (checked the code), and we never pass backslashes other than in the manifest path.

Reply:
Wait... there's a simpler solution here:

    cmd += ' -m "{}"'.format(manifest_path)

Quoting the path before splitting prevents the backslashes from getting removed.

Reply:
OK, nice suggestion. I will go for that one.

Reply:
@ulfalizer done